static/js/components/PrivacyForm.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+/* global SETTINGS: false */
+import React from 'react';
+
+import ProfileFormFields from '../util/ProfileFormFields';
+
+class PrivacyForm extends ProfileFormFields {
+  static propTypes = {
+    profile:        React.PropTypes.object,
+    ui:             React.PropTypes.object,
+    updateProfile:  React.PropTypes.func,
```

Comment:
would you mind switching this to a flow-style props type declaration? That would look like:

```js
import type { Profile } from '../flow/profileTypes';
import type { UIState } from '../reducers/ui';

class PrivacyForm extends ProfileFormFields {
  props: {
    profile:        Profile,
    ui:             UIState,
    updateProfile:  Function,
  };

  ...
}
```

this lets flow check the types for things on `this.props` properly, so it can know the type of, for instance, `profile.preferred_name`.